package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "action-verify-bytecode",
+  "version": "1.0.0",
+  "description": "Verify deployed contracts' bytecode",
+  "main": "dist/index.js",
+  "repository": "git@github.com:lidofinance/action-verify-bytecode.git",
+  "author": "madlabman <10616301+madlabman@users.noreply.github.com>",
```

Comment:
not sure if it really what you want to put this

Reply:
What's wrong with it?

Reply:
fans of this tool can't find you to send spam :(

Reply:
Ok, gave them a chance.